highfidelity/hifi
# Case 21893: Remove disabling of --url for Android

https://highfidelity.fogbugz.com/f/cases/21893/Enable-the-url-command-line-parameter-on-Android

# Testing
1.  Verify that build runs on Android
2.  Verify that build runs on Quest
3.  Verify build runs on PC
